src/pages/ProfilePage.tsx

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from 'react';
+import { makeStyles, Theme, createStyles } from '@material-ui/core'; // eslint-disable-line no-unused-vars
+import Avatar from '@material-ui/core/Avatar';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+
+import ProfileMenu from './ProfileMenu';
+
+
+const useStyles = makeStyles((theme: Theme) => createStyles({
+
+    container: {
+        backgroundColor: "#2d46b9",
+    },
+    marginAutoContainer: {
+        width: 500,
```

Comment:
I think the static containers make mobile sizing kinda wonky; but we can change later too if you'd like imo. Other than that, LGTM